src/DebugBar/Resources/debugbar.js

```diff
@@ -485,6 +491,7 @@ if (typeof(PhpDebugBar) == 'undefined') {
             }
 
             var self = this;
+            this.$el.css('display', 'block');
```

Comment:
this affects print mode, and in iframes
fixed on https://github.com/devzkhalil/php-debugbar/pull/3